arangod/IResearch/IResearchLinkMeta.cpp

```diff
@@ -172,10 +173,13 @@ bool IResearchLinkMeta::operator!=(IResearchLinkMeta const& other) const noexcep
   return meta;
 }
 
-bool IResearchLinkMeta::init(arangodb::velocypack::Slice const& slice, std::string& errorField,
-                             IResearchLinkMeta const& defaults /*= DEFAULT()*/,
-                             Mask* mask /*= nullptr*/
-                             ) noexcept {
+bool IResearchLinkMeta::init( // initialize meta
+    arangodb::velocypack::Slice const& slice, // definition
+    std::string& errorField, // field causing error (out-param)
+    IResearchLinkMeta const& defaults /*= DEFAULT()*/, // inherited defaults
+    TRI_vocbase_t const* defaultVocbase /*= nullptr*/, // fallback vocbase
+    Mask* mask /*= nullptr*/ // initialized fields (out-param)
+) noexcept {
```

Comment:
Technically, this method can throw exceptions, e.g. when initially constructing the static string object `fieldName`, when filling `errorField` etc.
So I suggest not declaring it `noexcept`.

Reply:
done